cea/interfaces/dashboard/api/contents.py

```
@@ -109,3 +122,231 @@
+    project: str
+
+
+VALID_EXTENSIONS = {".shp", ".dbf", ".prj", ".cpg", ".shx",
+                    ".csv", ".xls", ".xlsx",
+                    ".epw", ".tiff", ".tif", ".txt",
+                    ".feather"}
+
+def filter_valid_files(file_list: List[str]) -> List[str]:
+    return list(filter(lambda f: Path(f).suffix in VALID_EXTENSIONS, file_list))
+
+@router.post("/scenario/upload")
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)